### PR TITLE
Support delete vertex without edge

### PR DESCRIPTION
**Pull request by cangfengzhs**
#### What type of PR is this?
- [ ] bug
- [x] feature
- [ ] enhancement

#### What does this PR do?
support delete vertex without edge

#### Checklist：
- [x] Documentation affected （Please add the label if documentation needs to be modified.)
- [x] Incompatible （If it is incompatible, please describe it and add corresponding label.）
- [ ] Need to cherry-pick （If need to cherry-pick to some branches, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes：
Please confirm whether to reflect in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Comment by Shylock-Hg** [comment]
Why #3123  need this?

**Comment by cangfengzhs** [comment]
> Why #3123 need this?

It is part of the `support vertex with tag` feature mentioned in the review meeting

**Review by critical27 — APPROVED** [review]
LGTM

File: src/graph/validator/MutateValidator.cpp [hunk 1]
```diff
@@ -348,68 +350,71 @@ Status DeleteVerticesValidator::toPlan() {
 
   auto *dedupVid = Dedup::make(qctx_, nullptr);
   dedupVid->setInputVar(vidVar);
+  DeleteVertices *dvNode = nullptr;
+  if (withEdge_) {
+    std::vector<storage::cpp2::EdgeProp> edgeProps;
+    // make edgeRefs and edgeProp
+    auto index = 0u;
```
Comment (CPWstatic) [inline src/graph/validator/MutateValidator.cpp:357]: You could iterates both edgeTypes_ and edgeNames_ instead of using the index.

**Review by CPWstatic — APPROVED** [review]

File: src/graph/validator/MutateValidator.cpp (hunk 1 above)
Comment (cangfengzhs) [reply src/graph/validator/MutateValidator.cpp:357]: I did not modify these codes, and just move them into an if statement.

**Pull request merged**